app/models/github/issue.rb

```diff
@@ -7,6 +7,22 @@ class Github::Issue < ApplicationRecord
     class_name: "Github::IssueLabel",
     foreign_key: "github_issue_id"
 
+  scope :with_label, lambda { |label|
```

Comment:
The generated SQL looked good to me. Is this the right approach?

Reply:
Maybe 🤷 

Using `WHERE IN` instead of joins is often fine, because MySQL optimises. But you may well find you have a wider search space as you'll get a temporary table for each inner github_labels that then joins. It's definitely easier for the `not.in` bit below.

So yeah, I'd go with this and then we just check it in production once we're deployed. There's not really a huge amount of data in all this so it should be fine.